mantle/platform/qemu.go

```diff
@@ -742,13 +742,22 @@ func (builder *QemuBuilder) AddDisk(disk *Disk) error {
 }
 
 // AddInstallIso adds an ISO image
-func (builder *QemuBuilder) AddInstallIso(path string) error {
-	// We use bootindex=2 here: the idea is that during an ISO install, the
-	// primary disk isn't bootable, so the bootloader will fall back to the ISO
-	// boot. On reboot when the system is installed, the primary disk is
-	// selected. This allows us to have "boot once" functionality on both UEFI
-	// and BIOS (`-boot once=d` OTOH doesn't work with OVMF).
-	builder.Append("-drive", "file="+path+",format=raw,if=none,readonly=on,id=installiso", "-device", "ide-cd,drive=installiso,bootindex=2")
+func (builder *QemuBuilder) AddInstallIso(path string, bootindexStr string) error {
+	// Arches s390x and ppc64le don't support UEFI and use the cdrom option to boot the ISO.
+	// For all other arches we use ide-cd device with bootindex=2 here: the idea is
+	// that during an ISO install, the primary disk isn't bootable, so the bootloader
+	// will fall back to the ISO boot. On reboot when the system is installed, the
+	// primary disk is selected. This allows us to have "boot once" functionality on
+	// both UEFI and BIOS (`-boot once=d` OTOH doesn't work with OVMF).
+	switch system.RpmArch() {
+	case "s390x", "ppc64le":
```

Comment:
Could you please add aarch64. too. It seems to have the same issue.
```qemu-system-aarch64: -device ide-cd,drive=installiso,bootindex=2: No 'IDE' bus found for device 'ide-cd'```

Reply:
i did add it, but like we discussed `cdrom` will not work either. I'd like to get this in now so we can get the rhcos pipeline going for ppc64le